tests/pipeline/test_examples.py

```diff
@@ -32,7 +28,7 @@
 )
 def test_examples(example_module_name: str):
     example_module = importlib.import_module(f"examples.{dot_path_to_addon}.{example_module_name}")
-    if example_module_name.startswith("6"):
-        pipeline_utils.auto_run_pipeline(example_module.pipeline, wrapper=example_module.construct_webpage_by_response)
+    if example_module_name == "6_custom_messenger_interface":
+        check_happy_path(example_module.pipeline, HAPPY_PATH, ref_in_cand_comparer)
```

Comment:
a good idea

Reply:
Didn't get what ou mean actually. Mind providing an example?

Reply:
Wrap turns like it was before.
https://github.com/deeppavlov/dialog_flow_framework/blob/a1bf0e6167e826b0e6dfa14070620d961619d0fc/examples/pipeline/_pipeline_utils.py#L76

So that it would be:
`check_happy_path(example_module.pipeline, [(request, example_module.construct_webpage_by_response(response)) for request, response in HAPPY_PATH])`